_posts/2023-09-09-Margoliash-et-al-paper.md

```diff
@@ -40,6 +40,19 @@ A list of those STRs (if any) is available at the URL:
 
 https://margoliash-et-al-2023.s3.amazonaws.com/associations/{population}\_{phenotype}\_str\_gwas\_results.filtered.tab
 
+## SNP and indel association tests, by phenotype
+
+These files were generated using plink2.
+The columns are described [here](https://www.cog-genomics.org/plink/2.0/formats#glm_linear).
+The L100 and U100 columns are actually the lower and upper bounds of the 100\*(1-5e-8) confidence intervals,
+but that is rounded to 100 in the column name.
+
+URLs are of the form:  
+
+https://margoliash-et-al-2023.s3.amazonaws.com/associations/white\_british\_{phenotype}\_snp\_gwas\_results.tab.gz
```

Comment:
this link doesn't seem to be working for me

![image](https://github.com/user-attachments/assets/50c47125-bd6a-4ae6-85d8-e12f4c683577)

and neither is the one above it

Reply:
Does this wording change clarify things?

Reply:
ah oops, sorry - I didn't realize!